src/Microsoft.SqlTools.ServiceLayer/ObjectManagement/Requests/ProcessRenameEditRequest.cs

```diff
@@ -0,0 +1,23 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+using Microsoft.SqlTools.Hosting.Protocol.Contracts;
+using Microsoft.SqlTools.Utility;
+
+namespace Microsoft.SqlTools.ServiceLayer.ObjectManagement.Requests
+{
+    /// <summary>
+    /// Property class for ObjectManagement Service
+    /// </summary>
+    public class RenameRequestParams : GeneralRequestDetails
```

Comment:
1. Rename the `Requests` folder to `Contracts` (to match what we do elsewhere)
2. Rename this file to `RenameRequest.cs`